app/views/documents/_header.html.erb

```diff
@@ -7,10 +7,16 @@
   policies ||= nil
   specialist_tag_finder = SpecialistTagFinder.new(@document)
 %>
-<%= render partial: 'shared/heading',
-          locals: { type: add_sector_name(header_title, specialist_tag_finder.primary_sector_tag),
-                    heading: document.title,
-                    extra: true } %>
+<%=
+	render(
```

Comment:
could we remove the extra tab please?
It looks good and clean actually, but I'm sure it's not consistent with the rest of the codebase in Whitehall